### PR TITLE
Help migration

### PR DESCRIPTION
**Pull request by jburel**
Add troubleshooting entry for insight
migrating relevant content from https://help.openmicroscopy.org/troubleshooting.html and https://help.openmicroscopy.org/configure-client.html
cc @pwalczysko

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ - Mac/Linux: ``/yourhomedir/omero/log/omeroinsight.log``
+
+Submitting files
+----------------
+
+If an error occurred during import you can upload the files using the submission dialog.
+Automatic uploading of files to OME QA will not work if the file size is greater than 2GB.
+Submission of smaller file sizes can also not work if you are uploading over a slow or
+unreliable network. If the file upload to OME QA fails, please contact the team using our
+[support page](https://www.openmicroscopy.org/support/) to discuss alternative mechanism to submit the files.
```
Comment (mtbc) [inline FAQ.md:18]:
```suggestion
[support page](https://www.openmicroscopy.org/support/) to arrange an alternative mechanism to submit the files.
```

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+Adding third parties JARS
```
Comment (mtbc) [inline FAQ.md:20]:
```suggestion
Adding third parties' JARs
```

File: FAQ.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+ - Select ``Java > lib``.
+ - Drag and drop, or copy, the third-party ``.jar`` file into the ``lib`` folder.
+ - Start the application.
+
+**Windows**:
+
+ - Select the executable in Explorer.
+ - Select the ``lib`` folder.
+ - Drag and drop, or copy, the third-party ``.jar`` file into the ``lib`` folder.
+  - Start the application.
```
Comment (mtbc) [inline FAQ.md:47]:
```suggestion
 - Start the application.
```

File: FAQ.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+This will show you the location of the log file - ``omeroinsight.log``.
+If there are other log files with ``_1`` or ``_2`` appended to the name, ignore them.
+Alternatively you can find the log file in your user folder:
+ - Windows: ``C:\Users\you\omero\log\omeroinsight.log``
+ - Mac/Linux: ``/yourhomedir/omero/log/omeroinsight.log``
+
+Submitting files
+----------------
+
+If an error occurred during import you can upload the files using the submission dialog.
```
Comment (pwalczysko) [inline FAQ.md:14]:
```suggestion
If an error occurred during import you can upload the files using the submission dialog. Click on the ``Failed`` button next to the image whose import failed and then select ``Submit`` in the following menu. 
```

File: FAQ.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+[support page](https://www.openmicroscopy.org/support/) to arrange an alternative mechanism to submit the files.
+
+Adding third parties' JARs
+-------------------------
+
+OMERO.insight comes bundled with a number of standard Java Archives (JARs).
+However, functionality can be extended by adding JARs to OMERO.insight.
+An example of this is where some proprietary file formats require additional
+readers to enable OMERO to open and read the files. These readers can be provided
+by the owners of the formats as JARs, which need to be added to enable the functionality.
```
Comment (pwalczysko) [inline FAQ.md:27]:
```suggestion
by the owners of the formats as JARs, which need to be added to OMERO.insight to enable the functionality.
```

File: FAQ.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+Submission of smaller file sizes can also not work if you are uploading over a slow or
+unreliable network. If the file upload to OME QA fails, please contact the team using our
+[support page](https://www.openmicroscopy.org/support/) to arrange an alternative mechanism to submit the files.
+
+Adding third parties' JARs
+-------------------------
+
+OMERO.insight comes bundled with a number of standard Java Archives (JARs).
+However, functionality can be extended by adding JARs to OMERO.insight.
+An example of this is where some proprietary file formats require additional
```
Comment (pwalczysko) [inline FAQ.md:25]:
```suggestion
An example of this is when some proprietary file formats require additional
```

File: FAQ.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+**Linux**:
+
+ - Unzip the app.
+ - Go to the ``lib`` directory.
+ - Drag and drop, or copy, the third-party ``.jar`` file into the ``lib`` folder.
+ - Start the application.
+
+
+Choose Show Package Contents from the contextual menu.
```
Comment (pwalczysko) [inline FAQ.md:57]: Remove this sentence ? maybe a copy & paste error ?

File: FAQ.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+readers to enable OMERO to open and read the files. These readers can be provided
+by the owners of the formats as JARs, which need to be added to OMERO.insight to enable the functionality.
+
+If third-party JARs are related to the Bio-Formats import reader,
+they will also need to be added to the OMERO.server by your OMERO systems administrator.
+There is no need to do this if they are simply OMERO.insight update JARs.
+
+**Mac**:
+
+ - Right-click or control-click on on the omero.insight app.
```
Comment (pwalczysko) [inline FAQ.md:35]:
```suggestion
 - Right-click or control-click on the OMERO.insight app.
```

File: FAQ.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+ - Choose ``Show Package Contents`` from the contextual menu.
+ - Navigate to the ``Contents`` folder.
+ - Open the file  ``Java > OMERO.insight.cfg``.
+ - Edit the ``[JVMOptions]``section.
+
+**Windows**:
+
+You must add an optional parameter file to the same folder as the application.
+Create a file called ``OMERO.insight.vmoptions`` in the same directory as your OMERO.insight.exe file and
```
Comment (pwalczysko) [inline FAQ.md:81]:
```suggestion
Create a file called ``OMERO.insight.vmoptions`` in the same directory as your ``OMERO.insight.exe`` file and
```

**Comment by pwalczysko** [comment]
Re https://github.com/ome/omero-insight/pull/142/files#diff-04c6e90faac2675aa89e2176d2eec7d8R21 - is the ``FAQ.md`` also not belonging to the list of the contents of this directory ?

**Comment by pwalczysko** [comment]
Couple of edits, otherwise looks good.

**Pull request merged**